rln/src/utils.rs

```diff
@@ -173,6 +173,12 @@ pub fn bytes_be_to_vec_fr(input: &[u8]) -> Result<(Vec<Fr>, usize)> {
     Ok((res, read))
 }
 
+pub fn normalize_usize(input: usize) -> Vec<u8> {
+    let mut normalized_usize = input.to_le_bytes().to_vec();
+    normalized_usize.resize(8, 0);
+    normalized_usize
```

Comment:
You're still exporting as a `u64`. And I thought the point was to just depend on the architecture of the target machine. So that the ffi can rely on `usize` everywhere.

Reply:
As mentioned in https://github.com/vacp2p/zerokit/pull/94#discussion_r1121645204, we'd actually need to keep in mind that proofs that are generated by wasm32 need to have compatibility with other architectures :)